src/urllib3/util/ssltransport.py

```diff
@@ -2,9 +2,26 @@
 import socket
 import io
 
+from urllib3.exceptions import ProxySchemeUnsupported
+
 SSL_BLOCKSIZE = 16384
 
 
+def validate_ssl_context_for_tls_in_tls(ssl_context):
+    """
+    Raises a RuntimeError if the provided ssl_context can't be used for TLS in
+    TLS.
+
+    The only requirement is that the ssl_context provides the 'wrap_bio'
+    methods.
+    """
+
+    if not hasattr(ssl_context, "wrap_bio"):
```

Comment:
Maybe we should raise different error messages depending on the situation:
- If on Python 2.x: "TLS in TLS requires SSLContext.wrap_bio() which isn't supported on Python 2"
- If on Python 3.x: "TLS in TLS requires SSLContext.wrap_bio() which isn't available on non-native SSLContext"

Thoughts on this?

Reply:
It's a good idea, I like it and it will make the failure easier to understand. I'll make the corresponding changes.